src/amber/validations/params.cr

```diff
@@ -1,19 +1,19 @@
 module Amber::Validators
   # Holds a validation error message
-  record Error, param : String, value : Amber::Router::ParamsType, message : String
+  record Error, param : String, value : Amber::Router::Params, message : String
 
   # This struct holds the validation rules to be performed
   class BaseRule
-    getter predicate : (Amber::Router::ParamsType -> Bool)
+    getter predicate : (Amber::Router::Params -> Bool)
     getter field : String
-    getter value : Amber::Router::ParamsType
+    getter value = Amber::Router::Params.new
```

Comment:
Before I changed this I was getting an error about it not being set by initialize.